src/features/parameters/network-visualizations/network-visualizations-form.tsx

```
@@ -37,63 +37,37 @@ const useGetAvailableComponentLibraries = (user: User | null) => {
-                    item
-                    container
-                    direction="column"
-                    sx={{
-                        position: 'absolute',
-                        bottom: '30px',
-                    }}
-                >
-                    {renderActions?.()}
-                </Grid>
-            </Grid>
+            <ParameterLayout title={'NetworkVisualizations'} header={renderTitleFields?.()} actions={actions} isLoading={paramsLoading}>
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)